stories/PanZoom.stories.js

```diff
@@ -127,6 +128,24 @@ const PanZoomControlUI = (props) => {
   )
 }
 
+const AutoCenterDemo = () => {
+  const ref = useRef(null)
+  const [checked, setChecked] = useState(false)
+
+  return  (
+    <DefaultPanZoom ref={ref}>
+      <div style={{ padding: 20, width: 150, height: 100, backgroundColor: "red"}}>
+        <label>No animation <input style={{ display: "inline-block", }} type="checkbox" checked={checked} onChange={(e) => {
```

Comment:
It would be better to use the knobs for enabling / disabling the autoCenter animation as we are doing in the other stories, or have two buttons with animated / not animated autoCenter.

Reply:
done